script/install-lib-tensorflow-src/install.sh

```diff
@@ -0,0 +1,112 @@
+#! /bin/bash
+
+#
+# Installation script for RensorFlow library.
+#
+# See CK LICENSE.txt for licensing details.
+# See CK COPYRIGHT.txt for copyright details.
+#
+
+function stage() {
+  echo; echo "--------------------------------"; echo $1; echo
+}
+
+function exit_if_error() {
+ if [ "${?}" != "0" ]; then exit 1; fi
+}
+
+function remove_dir_if_exists() {
+  if [ -d $1 ]; then rm -rdf $1; fi
+}
+
+if [ "${PACKAGE_GIT}" == "YES" ] ; then
+  stage "Cloning package ${PACKAGE_URL}"
+  remove_dir_if_exists ${PACKAGE_SUB_DIR}
+  git clone ${PACKAGE_GIT_CLONE_FLAGS} ${PACKAGE_URL} ${PACKAGE_SUB_DIR}
+  exit_if_error
+
+  stage "Checking out branch ${PACKAGE_GIT_CHECKOUT}"
+  cd ${PACKAGE_SUB_DIR}
+  git checkout ${PACKAGE_GIT_CHECKOUT}
+  exit_if_error
+fi
+
+stage "Configure environment variables"
+# TF configure.py script will ask for all unset variables
+export CC_OPT_FLAGS="-march=native"
+export TF_NEED_S3=0
+export TF_NEED_GDR=0
+export TF_NEED_GCP=0
+export TF_NEED_HDFS=0
+export TF_NEED_OPENCL=0
+export TF_NEED_JEMALLOC=0
+export TF_NEED_VERBS=0
+export TF_NEED_MKL=0
+export TF_DOWNLOAD_MKL=0
+export TF_NEED_MPI=0
+export TF_CUDA_CLANG=0
+export PYTHON_BIN_PATH=${CK_ENV_COMPILER_PYTHON_FILE}
+export PYTHON_LIB_PATH="$($PYTHON_BIN_PATH -c 'import site; print(site.getsitepackages()[0])')"
+
+if [ "$CK_TF_NEED_CUDA" == "YES" ] ; then  
+  export TF_NEED_CUDA=1
+  echo "CUDA enabled"
+else
+  export TF_NEED_CUDA=0
+  echo "CUDA disabled"
+fi  
+
+if [ "$CK_TF_ENABLE_XLA" == "YES" ] ; then
+  export TF_ENABLE_XLA=1
+  echo "XLA enabled"
+else
+  export TF_ENABLE_XLA=0
+  echo "XLA disabled"
+fi
+
+if [ ${TF_NEED_CUDA} == 1 ] ; then
+  stage "Configure environment variables for CUDA"
+  CUDA_CONFIG_OPTS="--config=cuda "
+  export TF_CUDA_COMPUTE_CAPABILITIES="3.5,5.2,6.1,6.2"
```

Comment:
Will this work on TX1 (5.3)?

Reply:
I've checked it on TX1 in https://github.com/dividiti/ck-tensorflow-private/issues/26

Reply:
Excellent. My concern was that it would fall back silently to the CPU (and hence have the same performance).

Reply:
Still, there's probably no harm in adding 5.3 to this list.